### PR TITLE
Temperature correction via entity service

### PR DESCRIPTION
**Pull request by steffenaxer**
This commit allows us to set the temperature correction via an entity service, similar to the cover device.
You can test service by the help of the developer tools --> freeathome.thermostat_temperature_correction --> select your climate device

### CODE REVIEW

File: freeathome/fah/const.py [hunk 1]
```diff
@@ -105,6 +105,9 @@
 PID_MEASURED_TEMPERATURE = 0x0130
 PID_HEATING_DEMAND = 0x014D
 
+# Thermostat parameters
+PARAM_TEMPERATUR_CORRECTION = 0x001B
```
Comment (EnricoBilla) [inline freeathome/fah/const.py:109]: Looks good to me, there is just one little typo here.
Haven't tested it locally tho

Comment (steffenaxer) [reply freeathome/fah/const.py:109]: Thanks, I will fix it right now.

**Comment by Tho85** [comment]
Looks good :+1:

In the long run I think it would be better to have two generic services: `freeathome.set_datapoint` and `freeathome.set_parameter`. There are so many possible parameters and datapoints that we can't cover all of them with custom services. The services could be called like this:

```yaml
service: freeathome.set_datapoint
data:
  entity_id: cover.kitchen
  datapoint: force_position
  value: open
```

or

```yaml
service. freeathome.set_parameter
data:
  entity_id: climate.kitchen
  parameter: temperature_correction
  value: 0.5
```

However, this is still too far away, so let's merge this PR :rocket:

**Comment by steffenaxer** [comment]
I might have one more question we should discuss. Will give more infos this evening.

**Pull request merged**